src/app/services/profile/profile.service.ts

```diff
@@ -104,9 +104,9 @@ export class ProfileService {
           text: 'BACK',
           role: 'cancel',
           cssClass: 'secondary-button',
-          handler: () => {
-            this.location.back()
-          }
+          // handler: () => {
```

Comment:
why has this been commented?

Reply:
so there is a confusion which needs clarity and it will be discussed with kiran as vinod as suggested